CHANGELOG.md

```diff
@@ -48,6 +50,7 @@
         name: my-config-map
         key: my-key
   ``` 
+* Cruise Control logging configuration does not contain `cruisecontrol.root.logger` property anymore. It has been replaced by `rootLogger.level`.
```

Comment:
I do not think this is enough. I assume that users with inline logging need to change this. But additionally, any users with external logging need to have completely new logging file and have it stored under a new `log4j2.properties` file.

Reply:
I adding some more text here, let me know what you think!